FAQ.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
```

Comment:
Don't make it a class component. Keep a stateless component for this.
Also, I believe Question and Answers can be two separate components and they can be picked up from constants.js or maybe make a new file called strings.js and read from there to keep things tidy.

Reply:
Also, place it under pages directory. It will directly get served from there.